windows/src/engine/kmcomapi/processes/package/kpinstallpackage.pas

```diff
@@ -274,6 +274,8 @@ procedure TKPInstallPackage.Execute(const FileName: string; Options: TKPInstallP
             try
               with TKPInstallVisualKeyboard.Create(Context) do
               try
+                // TODO: we should probably try and do this by the keyboard install instead so
+                // that we can deprecate the associatedKeyboard field in .kvk
```

Comment:
TODO: we should probably try and do this by the keyboard install instead so that we can deprecate the associatedKeyboard field in .kvk